languages/go/exercises/concept/time/.docs/after.md

```diff
@@ -0,0 +1,10 @@
+A [`Time`][time] in Go is a type describing a moment in time. The date and time information can be accessed, compared, and manipulated through its methods. The current date and time can be retrieved through the [`time.Now`][now] function.
```

Comment:
This `after.md` is shorter than the `introduction.md`, which is unusual, but it does work well in this case. In general, whatever information is in the `introduction.md` document should also be in the `after.md` document. As such, you could consider including the bit of text on "To work with time, you will usually just call a method on the type you are working with, but there are also some methods called on the package itself." here too.